nin/frontend/app/scripts/utils/render.js

```diff
@@ -19,8 +18,13 @@ commands.on('stopRendering', () => {
 
 function render(i) {
   i = i || 0;
-  currentFrame = i;
   demo.jumpToFrame(i);
+  const currentFrame = demo.getCurrentFrame();
+  if (currentFrame < i) {
```

Comment:
The normal case would be that `currentFrame == i`, no? So that would stop rendering at the first frame.